<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EDispalyBoardForm</class>
 <widget class="QWidget" name="EDispalyBoardForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="TitleBar" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QWidget" name="widget_Down" native="true">
      <layout class="QGridLayout" name="gridLayout" columnstretch="1,1">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1">
         <item>
          <widget class="ConnectForm" name="widget_LeftUpConnectForm" native="true">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="SingleDMJForm" name="widget_LeftDownSingleDMJForm" native="true"/>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1">
         <item>
          <widget class="MulDMJForm" name="widget_RightUpMulDMJForm" native="true">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CurveForm" name="widget_RightDownCurveForm" native="true">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleBar</class>
   <extends>QWidget</extends>
   <header location="global">titlebar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConnectForm</class>
   <extends>QWidget</extends>
   <header location="global">connectform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MulDMJForm</class>
   <extends>QWidget</extends>
   <header location="global">muldmjform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SingleDMJForm</class>
   <extends>QWidget</extends>
   <header location="global">singledmjform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CurveForm</class>
   <extends>QWidget</extends>
   <header location="global">curveform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
